src/components/calculator.js

```diff
@@ -1,36 +1,61 @@
-          <button type="button" className="btn btnaction">&#247;</button>
-          <button type="button" className="btn">7</button>
-          <button type="button" className="btn">8</button>
-          <button type="button" className="btn">9</button>
-          <button type="button" className="btn btnaction">&#215;</button>
-          <button type="button" className="btn">4</button>
-          <button type="button" className="btn">5</button>
-          <button type="button" className="btn">6</button>
-          <button type="button" className="btn btnaction">-</button>
-          <button type="button" className="btn">1</button>
-          <button type="button" className="btn">2</button>
-          <button type="button" className="btn">3</button>
-          <button type="button" className="btn btnaction">+</button>
-          <button type="button" className="btnZero">0</button>
-          <button type="button" className="btn">.</button>
-          <button type="button" className="btn btnaction">=</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="AC">AC</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="+/-">+/-</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="%">%</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="&#247;">&#247;</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="7">7</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="8">8</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="9">9</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="&#215;">&#215;</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="4">4</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="5">5</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="6">6</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="-">-</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="1">1</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="2">2</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value="3">3</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="+">+</button>
+          <button onClick={this.handleClicks} type="button" className="btnZero" value="0">0</button>
+          <button onClick={this.handleClicks} type="button" className="btn" value=".">.</button>
+          <button onClick={this.handleClicks} type="button" className="btn btnaction" value="=">=</button>
```

Comment:
Well done 🔥 You implement all calculator operations with the click event 👌

Reply:
Thank you